dockertest/images.py

```diff
@@ -92,6 +92,10 @@ def __init__(self, repo, tag, long_id, created, size,
         self.user = user
 
         self.short_id = long_id[:12]
+        # docker-1.10 output includes a prefix describing the hash type
+        # FIXME: find canonical list
+        if long_id.startswith("sha256:"):
+            self.short_id = long_id[7:19]
```

Comment:
I suppose we can probably get at the hash-type another way, like in the JSON as well.  Okay, that's probably fine unless you think there's value to my comment above.